neuralforecast/common/_scalers.py

```diff
@@ -467,3 +467,83 @@ def inverse_transform(self, z, x_shift=None, x_scale=None):
     def forward(self, x):
         # The gradients are optained from BaseWindows/BaseRecurrent forwards.
         pass
+
+# %% ../../nbs/common.scalers.ipynb 38
+class RevIN(nn.Module):
```

Comment:
Given that we don't integrate RevIN as an actual scaler but keep it as a standalone module, I think it's best placed in `_modules` rather than `_scalers`